system-test/compute.js

```diff
@@ -40,60 +38,58 @@ describe('Compute', function() {
   // deleted. This will also pick up any previously-created objects that were
   // unable to be removed if a prior test run had unexpectedly quit.
 
-  const TESTS_PREFIX = 'gcloud-tests-';
+  const TESTS_PREFIX = 'gcloud';
+  const TEST_ID = uuid.v4().split('-')[0];
+  const FULL_PREFIX = `${TESTS_PREFIX}-${TEST_ID}`;
   const REGION_NAME = 'us-central1';
   const ZONE_NAME = 'us-central1-a';
 
   const compute = new Compute();
   const region = compute.region(REGION_NAME);
   const zone = compute.zone(ZONE_NAME);
 
-  before(deleteAllTestObjects);
-  after(deleteAllTestObjects);
+  const computeRequest = promisify(compute.request).bind(compute);
+  const zoneRequest = promisify(zone.request).bind(zone);
 
-  describe('addresses', function() {
+  before(() => deleteAllTestObjects({expiredOnly: true}));
+  after(() => deleteAllTestObjects({expiredOnly: false}));
+
+  describe('addresses', async () => {
     const ADDRESS_NAME = generateName('address');
     const address = region.address(ADDRESS_NAME);
 
-    before(create(address));
```

Comment:
LOL 😓

Reply:
This works (or at least it used to) :) It returns a function that execurew when the before hook fires.